Sprint-2/implement/querystring.js

```diff
@@ -14,3 +14,5 @@ function parseQueryString(queryString) {
 }
 
 module.exports = parseQueryString;
+
+console.log(parseQueryString("equation=x=y+1"));
```

Comment:
The given function is not implemented correctly. You are expected to fix it.

Here are some tests which you can try to see if the function returns the expected value:
```
parseQueryString("a=b=c=d=e")
parseQueryString("a=b&=&c=d")
parseQueryString("a=")
parseQueryString("=b")
parseQueryString("a=b&&c=d")
parseQueryString("a%25b=c%26d`)
```
Note: the `%25` and `%26` are ***URL encoded*** or ***percent encoded*** characters.